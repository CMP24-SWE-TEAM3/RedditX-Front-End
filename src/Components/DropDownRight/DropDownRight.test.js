import {shallow} from 'enzyme';

// import Components
import DropDownRightButton from "./DropDownRight";

describe("Drop down right Component", () => {
    it("this is a test for Drop down Component", () => {
        expect(shallow(<DropDownRightButton/>)).toMatchSnapshot();

    });
});