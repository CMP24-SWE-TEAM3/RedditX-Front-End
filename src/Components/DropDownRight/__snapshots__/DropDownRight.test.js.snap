// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Drop down right Component this is a test for Drop down Component 1`] = `ShallowWrapper {}`;
